plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -284,6 +284,18 @@ export class SessionRecordingIngester {
         gaugeSessionsRevoked.reset()
 
         const { team_id, session_id } = event
+
+        // added for incident 367 https://posthog.slack.com/archives/C08LPFEAUNR
+        // all session ids should be UUIDv7 and so 32 characters long
+        // we're now dropping invalid UUIDs at capture
+        // but have some naughty ones already in kafka
+        // this is a sanity check to get past those
+        if (session_id.length > 124) {
+            const semiValidTeamId = typeof team_id === 'number' || typeof team_id === 'string' ? team_id : 'unknown'
+            dropEvent(`session_id_too_long_${semiValidTeamId}`)
```

Comment:
wouldn't normally put team id here, but we're deleting this later today, and we're expecting a single value here